server/src/country.js

```diff
@@ -6,10 +6,23 @@ const route = express.Router();
 const cache = new NodeCache();
 const db = new Database('./countries.sqlite3');
 
+function getDate() {
+  return new Date().toISOString().split('T')[0];
+}
+
 route.get('/check', (req, res) => {
+  const cacheKey = 'solution';
+  let solution = cache.get(cacheKey);
+  const solutionNotAvailiable = solution === undefined;
```

Comment:
bare en liten skrivefeil i variable navn

```suggestion
  const solutionNotAvailable = solution === undefined;
```